### PR TITLE
Frontend

### PR DESCRIPTION
**Pull request by ctlong**
- Cleaned up and updated existing html and css.
- Initialized several angular controller/template pairs.

### CODE REVIEW

File: app/assets/javascripts/angular/controller/forms.js.erb [hunk 1]
```diff
@@ -1,3 +1,26 @@
-app.controller('ListFormsCtrl', ['$scope', function($scope){
+app.controller('ListFormsCtrl', ['$scope', '$http','$auth','$location', function($scope, $http, $auth, $location){
   
+}]);
+
+app.controller('FormsCreateCtrl', ['$scope', '$http','$auth','$location', function($scope, $http, $auth, $location){
+  
+  $scope.add = function(elem) {
+    var html = '';
```
Comment (h8rry) [inline app/assets/javascripts/angular/controller/forms.js.erb]: Why are you using this method to append content html? You are using Angular.js!

File: app/assets/javascripts/angular/controller/forms.js.erb [hunk 2]
```diff
@@ -1,3 +1,26 @@
...
+    html += '<li><a href="/#/forms/create">'
+    html +=  elem
+    html += '</a></li>'
+    $('#formStuff').append(html);
```
Comment (h8rry) [inline app/assets/javascripts/angular/controller/forms.js.erb]: Please don't mix jQuery and Angular together unless absolutely necessary

File: app/assets/javascripts/angular/controller/forms.js.erb [hunk 3]
```diff
@@ -1,3 +1,26 @@
...
+  };
+
+}]);
+
+app.controller('FormsUpdateCtrl', ['$scope', '$http', '$auth', '$location', '$routeParams', function($scope, $http, $auth, $location, $routeParams){
+  
+  url = "<%= ENV['URL'] %>" || 'http://localhost:3000'
+
+  $http.get(url + '/forms/' + $routeParams.id).success(function(response) {
+    $scope.name = response.name;
```
Comment (h8rry) [inline app/assets/javascripts/angular/controller/forms.js.erb]: Maybe it's better to do `$scope.form = response`

Then you can do

`$scope.form.name` and `$scope.form.content`

File: app/assets/javascripts/angular/controller/navbar.js.erb [hunk 4]
```diff
@@ -1,3 +1,18 @@
-app.controller('NavbarCtrl', ['$scope', '$location', function($scope, $auth, $location){
+app.controller('NavbarCtrl', ['$scope', '$auth', '$location','$window', function($scope, $auth, $location, $window){
 
+  $auth.validateUser().then(function(resp) {
```
Comment (h8rry) [inline app/assets/javascripts/angular/controller/navbar.js.erb:3]: Can you do `then()` on `validateUser()`?

File: app/assets/javascripts/application.js [hunk 5]
```diff
@@ -9,10 +9,11 @@
...
 // about supported directives.
+//= require jquery
 //= require angular/angular
 //= require angular-route/angular-route
 //= require bootstrap/dist/js/bootstrap
 //= require flat-ui/dist/js/flat-ui
-//= require angular-cookie/angular-cookie.min.js
-//= require ng-token-auth/dist/ng-token-auth.min.js
+//= require ng-token-auth/dist/ng-token-auth
+//= require angular-cookie/angular-cookie
```
Comment (h8rry) [inline app/assets/javascripts/application.js:18]: Good

File: app/assets/javascripts/application.js [hunk 6]
```diff
@@ -9,10 +9,11 @@
 //
 // Read Sprockets README (https://github.com/rails/sprockets#sprockets-directives) for details
 // about supported directives.
+//= require jquery
```
Comment (h8rry) [inline app/assets/javascripts/application.js:12]: Why do you need this

File: app/assets/templates/forms/create.html [hunk 7]
```diff
@@ -0,0 +1,31 @@
+<navbar></navbar>
+<div class="wrap-dashboard">
+  <div class="container-fluid wrap">
+    <div class="row">
+      <div class="list-forms text-center">
+        <div class="col-xs-4">
+          <h4>Selections</h4>
+          <h6><a href="" ng-click="add('text')">Text</a></h6>
```
Comment (h8rry) [inline app/assets/templates/forms/create.html:8]: Consider using an array + ng-repeat if the list gets longer

**Comment by h8rry** [comment]
@ctlong Please review

File: app/assets/javascripts/angular/controller/forms.js.erb (hunk 1 above)
Comment (ctlong) [inline app/assets/javascripts/angular/controller/forms.js.erb]: No excuse, I have no idea why I did that.

File: app/assets/javascripts/application.js (hunk 6 above)
Comment (ctlong) [inline app/assets/javascripts/application.js:12]: Cause bootstrap

File: app/assets/javascripts/angular/controller/navbar.js.erb (hunk 4 above)
Comment (ctlong) [inline app/assets/javascripts/angular/controller/navbar.js.erb:3]: Thats the way we did it in JimmyThis, is it wrong?

**Comment by ctlong** [comment]
Updated, how is it now?

**Comment by ctlong** [comment]
Updated again, how about now?

**Comment by h8rry** [comment]
Looks good.

**Pull request merged**